src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -22,29 +24,17 @@ export type PageOptions = {
 };
 
 export class Page {
-    constructor(protected readonly underlyingPage: Puppeteer.Page, options?: PageOptions) {
+    constructor(protected readonly underlyingPage: Playwright.Page, options?: PageOptions) {
         function forceEventFailure(eventDescription: string): void {
             forceTestFailure(
-                `Puppeteer.Page '${underlyingPage.url()}' emitted ${eventDescription}`,
+                `Playwright.Page '${underlyingPage.url()}' emitted ${eventDescription}`,
             );
         }
 
         function serializeError(error: Error): string {
             return `[Error]{name: '${error.name}', message: '${error.message}', stack: '${error.stack}'}`;
         }
 
-        underlyingPage.on('error', error => {
```

Comment:
are we able to remove this because of https://github.com/microsoft/playwright/blob/master/docs/api.md#event-crash? or is there a different reason?

Reply:
Yes, that's the replacement. Playwright doesn't actually support the `error` event on its `Page` objects, but even if it did we'd want to use the per-API exceptions instead for more local error reporting.